index.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+<meta charset="UTF-8" />
+<title>Timer!!!!</title>
+<link rel="stylesheet" type="text/css" href="styles.css">
+</head>
+<body>
+	<div id="app">
+		
+	</div>
+	
+	<script src="https://unpkg.com/react@15.3.2/dist/react.js"></script>
+	<script src="https://unpkg.com/react-dom@15.3.2/dist/react-dom.js"></script>
+	<script src="https://unpkg.com/babel-core@5.8.38/browser.min.js"></script>
+	
+	<script type="text/babel" src="script.js"></script>
```

Comment:
call compiled js and type change for javascript file

Reply:
now, it should be betrer:)